dev-libs/libpeas/libpeas-1.99.1.ebuild

```diff
@@ -0,0 +1,96 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+LUA_COMPAT=( lua5-1 )
+PYTHON_COMPAT=( python3_{10..12} )
+
+inherit gnome.org lua-single meson python-single-r1 vala virtualx xdg
+
+DESCRIPTION="A GObject plugins library"
+HOMEPAGE="https://wiki.gnome.org/Projects/Libpeas https://gitlab.gnome.org/GNOME/libpeas"
+
+LICENSE="LGPL-2.1+"
+SLOT="2"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~ia64 ~loong ~ppc ~ppc64 ~riscv ~sparc ~x86 ~amd64-linux ~x86-linux"
+
+IUSE="gjs glade +gtk gtk-doc lua +python vala"
+REQUIRED_USE="lua? ( ${LUA_REQUIRED_USE} )
+	python? ( ${PYTHON_REQUIRED_USE} )"
+
+RDEPEND="
+	>=dev-libs/glib-2.44:2
+	>=dev-libs/gobject-introspection-1.39:=
+	gjs? (
+		>=dev-libs/gjs-1.77.1
```

Comment:
I know that this matches the version requirement (`+gjs_req = '>= 1.77.1'`) but I suspect that we should actually require 1.77.2 because 1.77.2 is the first version that requires spidermonkey-115.